### PR TITLE
fix: regular runner names

### PR DESCRIPTION
**Pull request by bojiang**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/sklearn.py [hunk 1]
```diff
@@ -130,15 +130,11 @@ def __init__(
         batch_options: t.Optional[t.Dict[str, t.Any]],
         model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
     ):
+        tag = model_store.get(tag).tag
```
Comment (parano) [inline bentoml/sklearn.py:133]: let's put a hold on this PR and redo this change after https://github.com/bentoml/BentoML/pull/2108/ is merged

Comment (bojiang) [reply bentoml/sklearn.py:133]: sure

File: tests/e2e/bento_server_general_features/train.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from pickle_model import PickleModel
+
+import bentoml.sklearn
```
Comment (aarnphm) [inline tests/e2e/bento_server_general_features/train.py:3]: no need to do this with the recent changes

Comment (bojiang) [reply tests/e2e/bento_server_general_features/train.py:3]: actually i didn't import bentoml so it's needed.

Comment (aarnphm) [reply tests/e2e/bento_server_general_features/train.py:3]: oops my b

**Pull request merged**